config/default.json

```
@@ -612,7 +612,9 @@
     "sortByAttributes": {
       "Latest": "updated_at:desc",
       "Price: Low to high": "final_price",
-      "Price: High to low": "final_price:desc"
+      "Price: High to low": "final_price:desc",
+      "Position: First to last": "position:asc",
```

```suggestion
      "Position: First to last": "position",
```
`asc` is rather default one